xcube/cli/cli.py

```diff
@@ -12,52 +13,53 @@
 from xcube.cli.serve import serve
 from xcube.cli.timeit import timeit
 from xcube.cli.verify import verify
-from xcube.util.cliutil import parse_cli_kwargs, new_cli_ctx_obj, handle_cli_exception, cli_option_traceback, \
-    cli_option_scheduler
+from xcube.util.cliutil import cli_option_scheduler, cli_option_traceback, handle_cli_exception, new_cli_ctx_obj, \
+    parse_cli_kwargs
 from xcube.version import version
 
 
 # noinspection PyShadowingBuiltins
 @click.command(name="chunk")
-@click.argument('input', metavar='<input>')
-@click.argument('output', metavar='<output>')
-@click.option('--format', '-f', metavar='<format>', type=click.Choice(['zarr', 'netcdf']),
-              help="Format of the output. If not given, guessed from <output>.")
-@click.option('--params', '-p', metavar='<params>',
+@click.argument('cube')
+@click.option('--output', '-o', metavar='<OUTPUT>', default=DEFAULT_OUTPUT_PATH,
```

Comment:
```suggestion
@click.option('--output', '-o', metavar='OUTPUT', default=DEFAULT_OUTPUT_PATH,
```

And all the other too. See comment above.